docs/README.md

```diff
@@ -0,0 +1,17 @@
+# AWS C-S3 Client
+
+## Overview
+The AWS-C-S3 library is an asynchronous AWS S3 client focused on maximizing throughput and network utilization.
+
+### Key features:
```

Comment:
The Overview and Key features would be great in the main `aws-c-s3/README.md`. That's great stuff for everyone to know!

But without that, this `docs/README.md` would be pretty empty. It's fine if it's just links to other documents, or if we just move the contents of `GetObject.md` into here

Reply:
Thanks, I have merged this README into the main README. We can create a separate README in the `docs/` directory when we have more content.